samples/node-headless-ssr-proxy/config.js

```diff
@@ -141,25 +149,14 @@ const config = {
 
     if (cached) return Promise.resolve(cached);
 
-    return fetch(
-      `${config.apiHost}/sitecore/api/jss/dictionary/${appName}/${language}?sc_apikey=${
-        config.apiKey
-      }`,
-      {
-        headers: {
-          connection: "keep-alive",
-        },
-      }
-    )
-      .then((result) => result.json())
-      .then((json) => {
-        const viewBag = {
-          dictionary: json && json.phrases,
-        };
-
-        dictionaryCache.set(cacheKey, viewBag);
-        return viewBag;
-      });
+    return dictionaryService.fetchDictionaryData(language).then(phrases => {
+      const viewBag = {
+        dictionary: phrases,
+      };
+
+      dictionaryCache.set(cacheKey, viewBag);
```

Comment:
Our `RestDictionaryService` includes caching, so we should be able to get rid of this caching logic and the dependency on `node-cache`. Maybe we should explicitly set cache options (using same TTL value) in the `new RestDictionaryService` above to make it clear?

Reply:
Good catch! I forgot that we have default caching...